### DISCUSSION TITLE
Trying to set up different automatic builds that trigger off of different tags -> 'Build is not started because tag creation builds are disabled'

### DISCUSSION THREAD

**Discussion by bradv123** [codemagic-ci-cd/codemagic-docs#1604] [Q&A] [answered]
What I used to do was have a build trigger off any tag creation, it would build, create a release in github with auto generated notes, upload to google play internal track, and IOS test flight. This worked but it made testing a little more annoying. I wanted to change it so if a tag starts with 'dev' or 'android' i could have it only build without creating a release in github. 

The problem im now running into is when i push a new 'android' tag to github the build isnt starting. in the webhook settings of my project in codemagic i can see it says 'Build is not started because tag creation builds are disabled'. i dont understand why its saying that since that workflow im trying to get to trigger has "Trigger on tag creation" toggled on. is there a global setting to control this somehow?

Thanks

**Comment by icarusdust** [comment, marked as answer]
Hi @bradv123, have you defined `tag` event in the triggering section of your workflow settings ? https://docs.codemagic.io/yaml-running-builds/starting-builds-automatically/#build-triggers

**Reply by bradv123** [reply]
I had actually already solved my problem. I was getting confused with which build I was trying to have run. I had previously only ever used 1 build config so when I added more for my different build types I think I messed up the settings in one of them. I think also part of my problem too was the matching pattern. Many of the patterns that I tried didn't work, so I ended up using 'Android_v*' instead of trying to match 'Android_v1.0.5+56'. Thank you for your reply, I should have marked this as answered after I figured out my rookie mistake
